src/components/Statistic/Statistic.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+class Statistic extends React.Component {
+
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
+        return <p><span className="card-key">{this.props.type}: </span><span className="card-value">{this.props.details}</span></p>
```

Comment:
I'd probably split this out into a few more lines for readability

Reply:
on it